src/Libraries/Display/Display.cs

```diff
@@ -272,6 +348,41 @@ private static int ComputeSamplesFromNormalizedValue(double value, int lowestPow
             return finalExp;
         }
 
+        private static void CreateVertexColoredMesh(Point[] vertices, Color[] colors, IRenderPackage package, TessellationParameters parameters)
+        {
+            package.RequiresPerVertexColoration = true;
+
+            for (var i = 0; i <= vertices.Count()-3; i+=3)
+            {
+                var ptA = vertices[i];
+                var ptB = vertices[i+1];
+                var ptC = vertices[i+2];
+
+                var cA = colors[i];
+                var cB = colors[i+1];
+                var cC = colors[i+2];
+
+                var s1 = ptB.AsVector().Subtract(ptA.AsVector()).Normalized();
```

Comment:
should we add a note somewhere in the points tooltip about winding order? Will a flipped normal change the display of the mesh color / lighting calculation?